### ISSUE TITLE
Comprehensive List when license is needed

### ISSUE THREAD

**Issue by joergrenn** [MicrosoftDocs/microsoft-365-docs#5631]
As a customer I am having a hard time to figure out when a Shared mailbox needs a license.
Can you provide, or refer to ,a comprehensive list when a shared mailbox in O365 needs a license.
The note on this page saying:
> If you want to apply advanced features such as ... the shared mailbox must be licensed for those features.
Is too unclear.

Could you also clarify what means: "automatic" retention policies
Where is the difference between an automatic retention policy and a (regular) retention policy I can setup in exchange admin center or compliance center?

thanks.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 289f2440-9c44-7d93-20e3-baa9e9892cd9
* Version Independent ID: abc4e209-a071-d28a-cb01-aa72bbaee7c0
* Content: [About shared mailboxes - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/email/about-shared-mailboxes?view=o365-worldwide)
* Content Source: [microsoft-365/admin/email/about-shared-mailboxes.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/email/about-shared-mailboxes.md)
* Service: **o365-administration**
* GitHub Login: @SKjerland
* Microsoft Alias: **sharik**

**Comment by msbemba** [comment]
@joergrenn Thank you for your feedback.

1) Shared mailbox (without a license) has a default mailbox size of 50GB, If you would like to increase this limit an Exchange online plan 2 license is required.
2) Microsoft Defender for Office 365, Advanced eDiscovery, or automatic retention policies are a part of the subscription. So If the shared mailbox needs to be used for any of these features you need to license them separately as you do with a user mailbox.
3) Automatic retention labels are used in organizations where they would like to apply them automatically to content that matches specified conditions. Unlike Manual retention labels wherein the users/Admins will have to be trained on different classification labels available and when to apply them. For more information about Automatic retention labels you can visit - https://docs.microsoft.com/en-us/microsoft-365/compliance/apply-retention-labels-automatically?view=o365-worldwide

In a nutshell, when you don't assign a license to a shared mailbox you still have it protected with Exchange online protection, however, you cannot apply the advanced features such as defender, Advanced discovery or automatic retention policies.

Hope this helps!
Thanks
Sri

**Comment by joergrenn** [comment]
Hello @msbemba,
thanks for your reply.
Everywhere I look or ask I am told some things that require a license.
What a am looking for is not a summary in a nutshell, I am looking for a comprehensive and complete list of all features that require a shared mailbox to have a license.
best regards
Joerg

**Comment by msbemba** [comment]
@joergrenn Thank you for the response. The reason why you don't have a complete list is that it is no different from a user mailbox license requirement. A shared mailbox comes with a 50GB mail size, for anything and everything beyond that you need a license. You can refer to -https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-plan-options for all licensing options that are available in M365.

Hope this helps!

Thanks
Sri

**Comment by joergrenn** [comment]
@msbemba  sorry but it is not true that it is not different from a user mailbox license requirement.
Lets just take advanced eDiscorvery as an example. On the documentation https://docs.microsoft.com/en-us/microsoft-365/compliance/overview-ediscovery-20?view=o365-worldwide#subscriptions-and-licensing
> Licensing for Advanced eDiscovery requires the appropriate organization subscription and per-user licensing.
It says per user licensing, no mention of a shared mailbox licensing requirement.
This documentation is really not well aligned and does not make it easy to figure out which licenses are needed.

**Comment by msbemba** [comment]
@joergrenn  The shared mailbox needs the same licensing as a user mailbox to use any of the features. That is why I  said it is no different than a user mailbox. Whatever license you would assign a user mailbox, you will have to assign the same license for a shared mailbox to use the feature.

Thanks
Sri

**Comment by joergrenn** [comment]
@msbemba  nevermind, I should stop trying to help improve MS documentation.
